calico-enterprise/networking/egress/external-network.mdx

```diff
@@ -122,107 +122,49 @@ EOF
 
 ### Create egress gateways
 
-Create an egress gateway deployment running on the egress nodes and use new annotation `egress.projectcalico.org/externalNetworkNames` to associate the egress gateway pods of the deployment with one or more networks as desired. [see our other egress gateway guides](./)
+Use an Egress Gateway custom resource to deploy a group of egress gateways, and use `externalNetworks` to associate the egress gateway pods of the deployment with one or more networks as desired. [see our other egress gateway guides](./)
```

Comment:
s/Egress Gateway/egress gateway

We do not capitalize egress gateway. Please make this change throughout.